crates/astria-cli/src/cli/rollup.rs

```diff
@@ -0,0 +1,236 @@
+    /// Create a new rollup config
+    Create(ConfigCreateArgs),
+    /// Edit a rollup config
+    Edit(ConfigEditArgs),
+    /// Delete a rollup config
+    Delete(ConfigDeleteArgs),
+}
+
+#[derive(Args, Debug, Serialize)]
+pub struct ConfigCreateArgs {
+    #[clap(long, env = "ROLLUP_USE_TTY")]
+    pub use_tty: bool,
+    #[clap(long, env = "ROLLUP_LOG_LEVEL")]
+    pub log_level: String,
+
+    // rollup config
+    #[clap(long = "rollup.name", env = "ROLLUP_NAME")]
+    /// The name of the rollup
+    pub name: String,
+    #[clap(long = "rollup.chain-id", env = "ROLLUP_CHAIN_ID", required = false)]
+    /// Optional. Will be derived from the rollup name if not provided
+    pub chain_id: Option<String>,
+    #[clap(long = "rollup.network-id", env = "ROLLUP_NETWORK_ID")]
+    pub network_id: u64,
+    #[clap(long = "rollup.skip-empty-blocks", env = "ROLLUP_SKIP_EMPTY_BLOCKS")]
+    pub skip_empty_blocks: bool,
+
+    #[clap(long = "rollup.genesis-accounts", env = "ROLLUP_GENESIS_ACCOUNTS", num_args = 1..)]
+    pub genesis_accounts: Vec<GenesisAccountArg>,
+
+    // sequencer config
+    #[clap(
+        long = "sequencer.initial-block-height",
+        env = "ROLLUP_SEQUENCER_INITIAL_BLOCK_HEIGHT",
+        required = false
+    )]
+    /// Optional. If not set, will be determined from the current block height of the sequencer
+    pub sequencer_initial_block_height: Option<u64>,
+    #[clap(long = "sequencer.websocket", env = "ROLLUP_SEQUENCER_WEBSOCKET")]
+    pub sequencer_websocket: String,
+    #[clap(long = "sequencer.rpc", env = "ROLLUP_SEQUENCER_RPC")]
+    pub sequencer_rpc: String,
+}
+
+/// `GenesisAccount` is a struct that represents a genesis account to be funded.
+/// It has the form of `address:balance`.
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct GenesisAccountArg {
+    pub address: String,
+    pub balance: u128,
+}
+
+impl FromStr for GenesisAccountArg {
+    type Err = eyre::Report;
+
+    /// Parse a string of the form `address:balance` into a `GenesisAccountArg`.
+    /// If the balance is not provided, it will default to 0.
+    ///
+    /// # Errors
+    ///
+    /// * If the address is missing
+    /// * If the address is empty
+    /// * If the address is not a valid hex string that decodes to 20 bytes
+    fn from_str(s: &str) -> eyre::Result<Self, Self::Err> {
+        let mut parts = s.splitn(2, ':');
+
+        let address = parts.next().ok_or_else(|| eyre::eyre!("Missing address"))?;
+        let address = strip_0x_prefix(address).to_string();
+        if address.is_empty() {
+            return Err(eyre::eyre!("Empty address"));
+        }
+        let decoded =
+            hex::decode(&address).map_err(|e| eyre::eyre!("Invalid hex address: {}", e))?;
+        if decoded.len() != 20 {
+            return Err(eyre::eyre!(
+                "Address must be a 20-byte hex string, or 40 characters."
+            ));
+        }
+
+        let balance_str = parts.next().unwrap_or("0");
```

Comment:
Should we set a default balance if none is specified?

Reply:
The `unwrap_or` here is defaulting it to 0. Should it be 1000000000000000000000?

Reply:
That's kinda my thinking here, like have it default to at least the equivalent of 1 ETH, which would be `1000000000000000000`